.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
```

Comment:
Just double-checking... the "comments" inside this JSON are ok?

Reply:
Yes, this is based off the default config generated when you run VSCode's `Configure Workspace Recommended Extensions`. 

Opened a project with the same config as the PR and it does work resulting in this toast if you're curious what it looks like.

![image](https://user-images.githubusercontent.com/10877317/76579762-c4a7ce80-64a3-11ea-9931-6a6c9ae59385.png)